libraries/Microsoft.Bot.Schema/ContentTypes.cs

```diff
@@ -58,4 +58,11 @@ public partial class VideoCard
     {
         public const string ContentType = "application/vnd.microsoft.card.video";
     }
+
+    /// <summary>
+    /// OAuthCard ContentType value
+    public partial class OAuthCard
+    {
+        public const string ContentType = "application/vnd.microsoft.card.oauth";
```

Comment:
Useful code comments. :)

Reply:
this is good enough though right? it's pretty much just defining a constant (contenttype) for this class).

---
In reply to: [184823915](https://github.com/Microsoft/botbuilder-dotnet/pull/520#discussion_r184823915) [](ancestors = 184823915)